src/nuget/xunit.console.netcore.nuspec

```diff
@@ -33,11 +33,21 @@
       <dependency id="System.Threading" version="4.0.10" />
       <dependency id="System.Threading.Tasks" version="4.0.10" />
       <dependency id="System.Xml.XDocument" version="4.0.10" />
+      <dependency id="xunit.runner.utility" version="2.1.0-rc1-build3168" />
     </dependencies>
   </metadata>
   <files>
     <!-- xunit runner for .NET Core -->
     <file src="xunit.console.netcore\xunit.console.netcore.exe" target="lib\aspnetcore50" />
```

Comment:
@ericstj The aspnetcore50 moniker is currently still in use in [tests.targets](https://github.com/dotnet/buildtools/blob/65d93460c83bdcabfc1c0e53ff74f1f344cafa47/src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets#L47)